### DISCUSSION TITLE
Loading always active when using bundler

### DISCUSSION THREAD

**Discussion by flexchar** [livewire/livewire#6933] [7. Bugs (archived)]
### Livewire version

3 (this is also an issue on version 2)

### Which PHP version are you using?

PHP 8.2

### Repository URL

Not sure if this requires an example, but if so, I can make one.

### Steps To Reproduce

It appears that a loading indicator is always active `wire:loading` if there is an active websocket connection which in my case is Hot reloading offered by Vite library that I use to bundle tailwindcss. It's kinda a deal breaker since this means I cannot develop loading logic together with styling. :(

![image](https://github.com/livewire/livewire/assets/9198896/0099f4c8-d640-42ec-9d06-da46866f3c82)

If I block the Vite request, the issue is gone.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@flexchar just to confirm, do you have Livewire's styles in your app layout? If not, you should still add Livewire's styles manually to the layout when using Livewire's scripts in a bundle.

Hope this helps!

**Reply by flexchar** [reply]
Yes, they come automatically. 

```html
<!-- Livewire Styles --><style >[wire\:loading], [wire\:loading\.delay], [wire\:loading\.inline-block], [wire\:loading\.inline], [wire\:loading\.block], [wire\:loading\.flex], [wire\:loading\.table], [wire\:loading\.grid], [wire\:loading\.inline-flex] {display: none;}[wire\:loading\.delay\.shortest], [wire\:loading\.delay\.shorter], [wire\:loading\.delay\.short], [wire\:loading\.delay\.long], [wire\:loading\.delay\.longer], [wire\:loading\.delay\.longest] {display:none;}[wire\:offline] {display: none;}[wire\:dirty]:not(textarea):not(input):not(select) {display: none;}[x-cloak] {display: none;}</style>
```

**Reply by joshhanley** [reply]
@flexchar ah sorry, I misunderstood, I thought you were loading Livewire scripts in your Vite bundler, in which case Livewire styles don't get added automatically.

**Comment by bennettblack** [comment]
I'm also encountering this. Kind of surprised this one has slipped through, considering the popularity of Vite and hot-reloading.
